pkg/webhook/vlanconfig/validator.go

```diff
@@ -23,9 +23,11 @@ import (
 )
 
 const (
-	createErr = "could not create vlanConfig %s because %w"
-	updateErr = "could not update vlanConfig %s because %w"
-	deleteErr = "could not delete vlanConfig %s because %w"
+	createErr                           = "could not create vlanConfig %s because %w"
+	updateErr                           = "could not update vlanConfig %s because %w"
+	deleteErr                           = "could not delete vlanConfig %s because %w"
+	StorageNetworkNetAttachDefNamespace = "harvester-system"
```

Comment:
Please check if possible to refter this namespace definition
```
vendor/github.com/harvester/harvester/pkg/util/constants.go:	HarvesterSystemNamespaceName      = "harvester-system"
```

Reply:
sure added the changes